astroquery/esa/xmm_newton/tests/test_xmm_newton.py

```diff
@@ -12,6 +12,8 @@
 
 import pytest
 
+from astropy.coordinates import SkyCoord
+
 from ..core import XMMNewtonClass
 from ..tests.dummy_tap_handler import DummyXMMNewtonTapHandler
 
```

Comment:
Could you add a test to cover the `get_epic_metadata` function? `SkyCoord` is imported in the local tests file but the test itself was not included.

Reply:
Hi, sorry for the confusion. We initially thought about testing this function but since it basically makes use of the function query_xsa_tap we found the test was no longer needed. I should've deleted the import of SkyCoord. Please disregard this specific line of code; I'll deleted it and update the pull request.